backend/model/jobPoster.js

```diff
@@ -9,6 +9,7 @@ const {
   runJupyterLabServer,
   stopJupyterLabServer,
 } = require("../model/notebook");
+const { wallet } = require("../model/morphware");
```

Comment:
Not a dumb question, good question. Yeh so when wa import the wallet, it will create an instance of the MorphwareWallet here
https://github.com/morphware/daemon/blob/95c75ebd16fba284344ab799a348e1219f1e0071/backend/model/morphware.js#L171

Which as you said will initially create an instance of the Morphware wallet using the .add function

Reply:
Got it thanks!